docker-compose-coturn.yml

```diff
@@ -10,10 +10,18 @@ services:
     restart: unless-stopped
     ports:
       - "3000:3000"
+    environment:
+      - RTC_CONFIG=/home/node/app/rtc_config.json
+      - WS_FALLBACK=false # Set to true to enable websocket fallback if the peer to peer WebRTC connection is not available to the client.
+      - RATE_LIMIT=false # Set to true to limit clients to 1000 requests per 5 min.
+      - TZ=Europa/Vienna # Time Zone
+    #you need to copy rtc_config_example.json to rtc_config.json and specify domain, IP address, user and password
   coturn_server:
     image: "coturn/coturn"
     restart: always
     network_mode: "host"
     volumes:
       - ./turnserver.conf:/etc/coturn/turnserver.conf
+      - ./logs/:/var/log/
```

Comment:
as your logging to STDOUT now, you don't need that anymore, do you?

Reply:
this should be removed now.